### DISCUSSION TITLE
Is immutable elementary a dream or is it close to reality?

### DISCUSSION THREAD

**Discussion by telmobss1** [elementary/os#501] [Ideas]
well I could be talking about how good immutability would be in Elementarys because of the updates, but I wonder if people here would like Elementarys to be immutable

If possible I would like the developers to tell me if this is a reality or not

**Comment by cassidyjames** [comment]
It's definitely a long-term goal, and we're taking steps in that direction with things like moving our default apps to Flatpak. The direct work of building an immutable OS image has not been tackled by anyone at elementary yet, but I imagine we'd look into Endless OS as they were in a similar position (Ubuntu or Debian-based OS) and successfully converted their users over to an OS Tree based image.

**Comment by ryonakano** [comment]
We're working on this and I'm gonna close this one as a duplicate of https://github.com/elementary/os/issues/724.
